lib/api/owner.ts

```diff
@@ -22,6 +22,7 @@ export interface OwnerRes {
   publicKey?: string
   affiliate?: string
   trialEndsAt?: Date
+  blockAt?: Date
```

Comment:
Down below you can see this a few times:

```ts
parseDates<OwnerRes>(res, 'trialEndsAt')
```

You can add `blockAt` to it.

```ts
parseDates<OwnerRes>(res, 'trialEndsAt', 'blockAt')
```

Then `blockAt` will be parsed from an ISO-Date-String to a real Date

Reply:
Nice :+1: